app/javascript/components/verifiable.html

```diff
@@ -1,6 +1,15 @@
-<span>
-  <p>Is this shown at <a target="_blank" rel="noopener nofollow" class="external free" :href="page.reference_url">{{ page.reference_url }}</a>?</p>
+<div>
+  <span v-if="page.reference_url">
+    <p>Is this shown at <a target="_blank" rel="noopener nofollow" class="external free" :href="page.reference_url">{{ page.reference_url }}</a>?</p>
 
-  <button v-on:click="submitStatement(true)" class="mw-ui-button mw-ui-progressive">Yes</button>
-  <button v-on:click="submitStatement(false)" class="mw-ui-button">No</button>
-</span>
+    <button v-on:click="submitStatement(true)" class="mw-ui-button mw-ui-progressive">Yes</button>
+    <button v-on:click="submitStatement(false)" class="mw-ui-button">No</button>
+  </span>
+  <span v-else>
+    <p>
+      Please enter a reference URL:
+      <input v-model="referenceURL" type="url">
+      <button v-on:click="$emit('reference-url-change', referenceURL)">Save</button>
```

Comment:
I hadn't seen this approach to emitting events. Looks like it is emitting to a parent's function passed in as a prop which took me some time to get my head around but it works :)

Reply:
Yeah I hadn't seen it done like this before, but I spotted it in the docs for `$emit`: https://vuejs.org/v2/api/#vm-emit, so seems to be fairly standard, from what I can tell!

Reply:
Ah actually it was these docs I saw it in (they're roughly the same as the `$emit` docs, but passes an argument to the callback. https://vuejs.org/v2/guide/components.html#Emitting-a-Value-With-an-Event